test/libsolidity/syntaxTests/multiVariableDeclaration/oneElementTuple.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+    function f() public {
+        (uint a,) = (1,);
```

Comment:
I'm not sure whether the right hand side even needs to be well-formed in the future (once tuple declarations and assignments with mismatching numbers of components are disallowed).

Reply:
What do you mean by "well-formed"?